### PR TITLE
Remaining tweaks to General Tools

### PR DESCRIPTION
**Pull request by tomchadwin**
Minor language corrections/improvements

### CODE REVIEW

File: source/docs/user_manual/introduction/general_tools.rst [hunk 1]
```diff
@@ -1596,54 +1596,54 @@ to duplicate any layer in the map legend.
...
   Given that whenever you apply modifications to the layer properties, changes
   are stored in the active style, always ensure you are editing the right style
-  to avoid mistakenly alter a style used in a :ref:`map theme <map_themes>`.
+  to avoid mistakenly altering a style used in a :ref:`map theme <map_themes>`.
 
 .. tip:: **Manage styles from layer context menu**
 
-   Right-click on the layer in :guilabel:`Layers Panel` to add, rename
-   or remove layer style.
+   Right-click on the layer in the :guilabel:`Layers Panel` to add, rename
```
Comment (DelazJ) [inline source/docs/user_manual/introduction/general_tools.rst]:
```suggestion
   Right-click on the layer in the :guilabel:`Layers` panel to add, rename
```
Panel is no longer part of the GUI label

File: source/docs/user_manual/introduction/general_tools.rst [hunk 2]
```diff
@@ -1596,54 +1596,54 @@ to duplicate any layer in the map legend.
...
+in the database. Click on :menuselection:`Save Style --> Save in database`
 then fill in the dialog to define a style name, add a description, a :file:`.ui`
-file if applicable and check if the style should be the default style.
+file if applicable and to check if the style should be the default style.
 
-You can save several styles for a single table in the database. However each
-table can have only one default style. Default style can be saved in the layer
+You can save several styles for a single table in the database. However, each
+table can have only one default style. Default styles can be saved in the layer
 database or in the QGIS local database, a SQLite database in the :file:`~/.qgis2/`
```
Comment (DelazJ) [inline source/docs/user_manual/introduction/general_tools.rst]: another qgis2 we should remove. Sorry I don't know what's the replacement.

File: source/docs/user_manual/introduction/general_tools.rst [hunk 3]
```diff
@@ -1596,54 +1596,54 @@ to duplicate any layer in the map legend.
...
 
 Save in database
 .................
 
-Vector layer style can also be stored in a database if the layer datasource is a
+Vector layer styles can also be stored in a database if the layer datasource is a
 database provider. Supported formats are PostGIS, GeoPackage, SpatiaLite, MSSQL
 and Oracle. The layer style is saved inside a table (named :file:`layer_styles`)
-of the database. Click on :menuselection:`Save Style --> Save in database` item
+in the database. Click on :menuselection:`Save Style --> Save in database`
```
Comment (DelazJ) [inline source/docs/user_manual/introduction/general_tools.rst]:
```suggestion
in the database. Click on :menuselection:`Save Style... --> Save in database`
```
That's not the purpose of this PR but I'm afraid all this "Storing Styles in a File or a Database" section is outdated. eg there no more "Save in database" entry but you get a dialog in which you select either qml, sld, or db, with style's properties.

File: source/docs/user_manual/introduction/general_tools.rst [hunk 4]
```diff
@@ -1663,34 +1663,34 @@ directory (where QGIS stores its local settings).
...
-to any layer whatever its format is, loading styles stored in database is only
+:menuselection:`Style --> Load Style` helps you apply any saved style to a
+layer. While text-file styles (:file:`.sld` or :file:`.qml`) can be applied
+to any layer whatever its format, loading styles stored in a database is only
 possible if the layer is from the same database or the style is stored in the
 QGIS local database.
 
-The :guilabel:`Load Style from Database` dialog displays a list of related
-styles to the layer found in the database and all the other styles saved in it,
+The :guilabel:`Load Style from Database` dialog displays a list of styles related
```
Comment (DelazJ) [inline source/docs/user_manual/introduction/general_tools.rst]:
```suggestion
The :guilabel:`Database Styles Manager` dialog displays a list of styles related
```
While waiting for the section update let's use actual name

File: source/docs/user_manual/introduction/general_tools.rst [hunk 5]
```diff
@@ -1704,43 +1704,43 @@ project's title, or the user's full name) that can be used in expressions.
...
+example in print layout creating a label with this content:
 
   This map was made using QGIS [% @qgis_version %]. The project file for this
   map is: [% @project_path %]
 
 Will render the label like this::
 
-  This map was made using QGIS 2.14. The project file for this map is:
-  /gis/qgis-user-conference-2015.qgs
+  This map was made using QGIS 3.4.4. The project file for this map is:
```
Comment (DelazJ) [inline source/docs/user_manual/introduction/general_tools.rst]:
```suggestion
  This map was made using QGIS 3.4.4-Madeira. The project file for this map is:
```
Looks like qgis_version takes name now

File: source/docs/user_manual/introduction/general_tools.rst (hunk 2 above)
Comment (havatv) [reply source/docs/user_manual/introduction/general_tools.rst]: Location depends on the operating system.

Comment (DelazJ) [reply source/docs/user_manual/introduction/general_tools.rst]: I think the styles db is now in the QGIS user profile and while the location of the profile folder depends on OS, the path to the db in the folder is the same for all and mentioning user profile (if confirmed) here should be a minimum (in any case, we must not keep .qgis**2**)
User profiles folders are/should be described elsewhere.

File: source/docs/user_manual/introduction/general_tools.rst (hunk 3 above)
Comment (tomchadwin) [reply source/docs/user_manual/introduction/general_tools.rst]: Sure - a job for a separate PR, I guess.

File: source/docs/user_manual/introduction/general_tools.rst (hunk 2 above)
Comment (tomchadwin) [reply source/docs/user_manual/introduction/general_tools.rst]:
```suggestion
database.
```

File: source/docs/user_manual/introduction/general_tools.rst [hunk 6]
```diff
@@ -1596,54 +1596,54 @@ to duplicate any layer in the map legend.
...
 then fill in the dialog to define a style name, add a description, a :file:`.ui`
-file if applicable and check if the style should be the default style.
+file if applicable and to check if the style should be the default style.
 
-You can save several styles for a single table in the database. However each
-table can have only one default style. Default style can be saved in the layer
+You can save several styles for a single table in the database. However, each
+table can have only one default style. Default styles can be saved in the layer
 database or in the QGIS local database, a SQLite database in the :file:`~/.qgis2/`
 directory (where QGIS stores its local settings).
```
Comment (tomchadwin) [inline source/docs/user_manual/introduction/general_tools.rst]:
```suggestion
```

File: source/docs/user_manual/introduction/general_tools.rst (hunk 2 above)
Comment (DelazJ) [reply source/docs/user_manual/introduction/general_tools.rst]: I's not the layer database.
We have two different things here when you press "save as default" button from a database layer; you get a pop-up dialog with options to save style to "datasource database", "local database" or "cancel". And the differences between the first two options need to be clarified.

**Comment by DelazJ** [comment]
Let's merge and address doc updates in a second step

**Comment by DelazJ** [comment]
Thanks @tomchadwin

**Pull request merged**